src/xpk/core/core.py

```diff
@@ -2104,8 +2104,12 @@ def get_main_container(args, system, docker_image, resource_type) -> str:
         ' is required but not installed. Aborting"; exit 24;};'
     )
     xpk_internal_commands += (
+        'set -x;'
```

Comment:
Curious why we add `set -x;` here? Is it related to the debug_dump_gcs feature or a more general feature? If it is a more general feature, maybe it should be part of --enable-debug-logs arg?